hijack/middleware.py

```diff
@@ -1,30 +1,52 @@
+import re
+
+from django.template.loader import render_to_string
 from django.utils.deprecation import MiddlewareMixin
 
+from hijack.conf import settings
 
-class HijackRemoteUserMiddleware(MiddlewareMixin):
-    """
-    Middleware for hijack RemoteUser.
+__all__ = ["HijackUserMiddleware"]
 
-    One must place this middleware between
-    'django.contrib.auth.middleware.AuthenticationMiddleware' and
-    'django.contrib.auth.middleware.RemoteUserMiddleware' in MIDDLEWARE_CLASSES
+_HTML_TYPES = ("text/html", "application/xhtml+xml")
 
-    Just makes remote user same as hijacked
-    """
 
-    header = "REMOTE_USER"
+class HijackUserMiddleware(MiddlewareMixin):
```

Comment:
middlewareMixin was used for the deprecation period/BC, so is using it still relevant?

Reply:
Is there a good alternative? Since we do not have any coroutine-specific code, I would probably end up writing a similar class. Besides, the `MiddlewareMixin` is not scheduled for deprecation.